src/sage/ext/fast_callable.pyx

```diff
@@ -508,7 +508,7 @@ def _builder_and_stream(vars, domain):
     - ``domain`` -- a Sage parent or Python type or ``None``; if non-``None``,
       all arithmetic is done in that domain
 
-    OUTPUT: A :class:`Wrapper`, an class:`InstructionStream`
+    OUTPUT: A :class:`Wrapper`, a class:`InstructionStream`
```

Comment:
this should be "an :class:" -- this is only Sphinx markup

Reply:
I agree.

Reply:
```suggestion
    OUTPUT: A :class:`Wrapper`, an class:`InstructionStream`
```